### DISCUSSION TITLE
求助：关于命令行提示未找到

### DISCUSSION THREAD

**Discussion by Nemoings** [Boos4721/openwrt#69] [Q&A/答疑解惑]
在SSH中成功刷入如下命令：
sysupgrade -F -n /tmp/openwrt-ipq807x-generic-xiaomi_ax3600-squashfs-nand-sysupgrade.bin
但是启动从过渡固件启动至qsdk时出错，根据大佬教程应该输入：
nvram set flag_last_success=1
nvram set flag_boot_rootfs=1
nvram commit
但是命令提示未找到，
PS.  非SC学科相关专业，写的乱了些，祝大佬新春愉快，可付费求教问题

**Comment by Boos4721** [comment]
**Telegram DM**
